src/components/Tool/MergeContacts/Contact.tsx

```diff
@@ -120,15 +159,15 @@ const Contact: React.FC<Props> = ({ contact1, contact2, update }) => {
                 <Box
                   display="flex"
                   alignItems="center"
-                  className={classes.contactBasic}
+                  className={`
+                  ${classes.contactBasic} 
+                  ${
+                    selected === 'left'
```

Comment:
`selected === 'left'` is used in mutiple locations, you could make it into a variable.
Same with `selected === 'right'`

Reply:
@dr-bizz what variable are you thinking? Like a `leftSelected` boolean variable?

Reply:
yes